.github/workflows/deploy.yml

```diff
@@ -25,8 +25,11 @@ jobs:
       - name: Pull down dependencies
         run: npm install
 
+      - name: Run unit tests
+        run: npm run test:ci
```

Comment:
is this "ci" a headless command? or e2e

Reply:
yea the `test:ci` script essentially runs the unit tests in chrome headless mode and exits once complete and doesn't watch files for changes

in package.json it runs `ng test --watch=false --browsers=ChromeHeadless --code-coverage=true`